### ISSUE TITLE
Can't add a position with a price lower than 1€

### ISSUE THREAD

**Issue by Norbert01** [premnirmal/StockTicker#188]
When I add a position with a stock price 0,50 the price change to 5,00 .

**Comment by premnirmal** [comment]
Can you try entering the price as 0**.**50 and not 0,50?

**Comment by Norbert01** [comment]
Hello,

I can't enter a "*". It dosen't work additional "0". I can only enter numbers 0-9 and "," .

Regards,
Norbert


Am 2. Februar 2021 13:03:50 MEZ schrieb Prem Nirmal <notifications@github.com>:
>Can you try entering the price as 0**.**50 and not 0,50?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/premnirmal/StockTicker/issues/188#issuecomment-771588451

-- 
Mobiltelefon

**Comment by Norbert01** [comment]
I tried c&p 0.5 --> dosen't work.

Am 2. Februar 2021 13:03:50 MEZ schrieb Prem Nirmal <notifications@github.com>:
>Can you try entering the price as 0**.**50 and not 0,50?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/premnirmal/StockTicker/issues/188#issuecomment-771588451

-- 
Mobiltelefon

**Comment by Dan-Mikkel** [comment]
Have the same trouble with my norwegian phone as well.

**Comment by Dan-Mikkel** [comment]
Found a workaround, which is not ideal, but it helps:
Write down the number somewhere else and paste it into the field.

**Comment by Dan-Mikkel** [comment]
> Can you try entering the price as 0.50 and not 0,50?

My keyboard does not allow me to write with 0,50, my only option is . when i write. Might be some localization issues with the keyboard.

**Comment by ThomasWaldmann** [comment]
Strange, I have no issue with this (just tried to reproduce):
- 1000 shares
- price 0,12
- correctly computes value to 120,00

**Comment by ThomasWaldmann** [comment]
Note: i used a comma to make it work. when entering price `0.12` it was wrongly interpreted as `12,00`.

**Comment by ThomasWaldmann** [comment]
Note: issue topic should be more general.

this is not a problem limited to prices lower than 1 EUR, but a general issue with the decimal separator.

**Comment by opk12** [comment]
Yes, this is a general issue with dot / comma and it should have a matching title. I am using AnySoftKeyboard, which has dot and comma, regardless of the locale.

* Set the USA locale in the system settings. In the field for `Alert above`, type `2.15` and save; reopen, it becomes `2,15`; save without changing; reopen, it becomes `215,00`.
* Set any of these locales in the system settings: French, German, Italian. Type `2.15` and save; reopen, it becomes `215,00`; save without changing, it stays the same.
